src/pages/Tailwind/Pool/PoolCardRight.tsx

```diff
@@ -58,7 +59,17 @@ const PoolCardRight = ({ pool }: PoolCardRightProps) => {
           <div className="font-fredoka text-4xl">{formatNumber(pool.held)} HLP</div>
         </div>
         <div className="flex items-end">
-          <PoolBigButton title="Stake it" isEnabled={pool.held > 0} onClick={stake} />
+          <PoolBigButton
+            title="Stake it"
+            isEnabled={pool.held > 0}
+            onClick={
+              isActivePool && pool.held > 0
```

Comment:
Suggestion: can we move this logic inside the `stake()` function instead?

Reply:
I think we also want to add another condition to redirect only if the button is enabled (e.g. `pool.held > 0`)